src/HomePage/Skills/Skills.scss

```diff
@@ -1,17 +1,29 @@
-.skills {
-  display: flex;
-  justify-content: space-around;
-  flex-flow: wrap;
-  max-width: 550px;
-}
+.skills-section {
+  background: rgb(219, 217, 217);
+  min-height: 100vh;
+  
 
-.icons {
-  font-size: 100px;
-  color: #ff4040;
-}
+  .skills {
+    display: flex;
+    justify-content: space-around;
+    flex-flow: wrap;
+    max-width: 800px;
+    align-content: center;
+  }
+
+  .icons {
+    font-size: 100px;
+    color: #c56565;
+    padding: 30px;
+  }
+
+  .wrapper {
+    display: flex;
+    justify-content: center;
+    height: 350px;
+  }
 
-.wrapper {
-  display: flex;
-  justify-content: center;
-  height: 250px;
+  .skills-intro {
+    font-family: "Roboto";
```

Comment:
This is more of a design comment but you seem to have a lot of fonts already. Not a big problem but its best to keep to 1/2 fonts across the entire site so i wouldn't add too many more

Reply:
Also this font doesnt actually work on my browser for some reason. Does it work on yours? Maybe test it out on different browsers and stuff and check it is properly working

Reply:
Yeah I kept changing the font to see if anything would work, but the font doesn't show for some odd reason....